index.html

```diff
@@ -193,23 +169,38 @@ <h3>OTC Talks #2</h3>
 					target="_blank"
 					rel="noreferrer"
 					>join our weekly online OTC CatchUp sessions</a
-				>.
-			</p>
-
-			<h2 id="past-talks">Past Talks</h2>
-
-			<div class="event">
+					>.
+				</p>
+				
+				<h2 id="past-talks">Past Talks</h2>
+				<div class="event">
+					<h3>OTC Talks #2</h3>
+					<div>19th February 2023, 10:30 PM IST</div>
+					<!-- <p>
+						Link to attend —
+						<a href="https://catchup.ourtech.community/attend"
+							>OTC CatchUp #119</a
+						>
+					</p> -->
+	
+					<p>
+						"Internals of Email Relay" by
+						<a
+							href="https://twitter.com/omtalk"
+							target="_blank"
+							rel="noreferrer"
+							>Omkar Khair</a
+						>
+					</p>
+	
+					<p>
+						<a href="/2.html">More details</a>
```

Comment:
Please remember to change this from `/2.html` to `/2`.

Reply:
Yes.